docs/lite.md

```diff
@@ -0,0 +1,43 @@
+# SC4S Lite
+## Purpose
+SC4S Lite has been designed to provide a scalable, performance-oriented solution for syslog data ingestion into Splunk. 
+The addition of Pluggable Modular Parsers offers users the flexibility to incorporate custom data processing logic to suit specific use cases.
+## Architecture
+![architecture diagram](sc4slite_arch_diag.png)
+
+## Components and Modules
+
+### SC4S Lite
+The primary component of the system, SC4S Lite is built upon the SC4S, providing a lightweight, high-performance sc4s  solution, the current SC4S is very complex and packed with too many oob parser support.
```

Comment:
The primary component of the system is SC4S Lite. It is a lightweight, high-performance solution build upon the default SC4S in order to reduce complexity and the number of pre-defined parsers.